Documentation/Game Design.txt

```diff
@@ -16,43 +16,77 @@ UI:
   => Map
     - Cursor selects planets
     - Arrow keys control ship
+
   => Current planet info
-    - Displays info when a ship collides with a planet on the map
+    - Displays planet info when a ship collides with a planet on the map
+
   => Selected planet info
-    - Displays info when a player hovers the cursor over a planet
+    - Displays planet info when a player hovers the cursor over a planet
```

Comment:
Do we want it to be hovers over, or clicks? I feel like hovering would be annoying

Reply:
Probably clicks, but we can always change our minds later

Reply:
For testing we can start with hovering, but I wanna change that before completion